domain/src/main/java/edu/nextstep/camp/calculator/domain/Calculator.kt

```diff
@@ -1,18 +1,30 @@
 package edu.nextstep.camp.calculator.domain
 
 class Calculator {
-    fun evaluatesExpression(expression: String?): Double{
-        if(expression.isNullOrBlank()) throw IllegalArgumentException("입력값이 null이거나 빈 공백 문자")
+    fun evaluatesExpression(expression: String?): Double {
```

Comment:
```suggestion
    fun evaluatesExpression(expression: Expression): Double {
```
Expression 클래스를 만드셨으니, 해당 클래스를 활용해보면 어떨까요?